src/lib/api/fetch.ts

```diff
@@ -0,0 +1,7 @@
+import { get } from './client'
+
+export async function fetcher(path: string) {
+  const response = await get(path)
+
+  return response
+}
```

Comment:
요기두 제네릭 타입 인자를 `get`한테 넘겨줘야 할 것 같아요!

```suggestion
export async function fetcher<T>(path: string) {
  const response = await get<T>(path)

  return response
}
```